### PR TITLE
Switch to python 3 variants of dependencies on debian-based builds

### PR DESCRIPTION
**Pull request by thaJeztah**

### CODE REVIEW

File: contrib/debian/dev-packages.lst [hunk 1]
```diff
@@ -4,7 +4,7 @@ libprotobuf-dev
 libprotobuf-c-dev
 protobuf-c-compiler
 protobuf-compiler
-python-protobuf
+python3-protobuf
 libnet-dev
 
 # Extra packages, required for testing and building other tools
```
Comment (thaJeztah) [inline contrib/debian/dev-packages.lst:10]: Not sure if the `python-ipaddr` package is needed if python 3 is used; https://packages.debian.org/buster/python-ipaddr

> This module was incorporated into python3.3 and later as ipaddress. A direct backport of ipaddress is available in Debian as python-ipaddress. There are incompatibilies between the two, so both are provided

File: contrib/debian/dev-packages.lst [hunk 2]
```diff
@@ -4,7 +4,7 @@ libprotobuf-dev
 libprotobuf-c-dev
 protobuf-c-compiler
 protobuf-compiler
-python-protobuf
+python3-protobuf
```
Comment (thaJeztah) [inline contrib/debian/dev-packages.lst:7]: This was the main reason I arrived here; I noticed that our Dockerfile installed `python-protubuf`, which depends on python 2, so I wondered if we could switch. Then I noticed there's other dependencies that still defaulted to python 2, so I updated those.

Haven't tested yet locally, so "abusing" CI on this repository to see if everything is ok with my change

**Comment by adrianreber** [comment]
Not able to take a close look, but this sounds similar to https://github.com/checkpoint-restore/criu/pull/1074

**Comment by thaJeztah** [comment]
Oh! Didn't see that one; yes, might be similar (also didn't look closely)

Looks like your PR needs a rebase (I think the dev branch has been reset at some point; I saw a similar issue on another PR)

**Comment by adrianreber** [comment]
@thaJeztah could you rebase this PR. The other PR switching CI to Python 3 has been merged.

**Comment by thaJeztah** [comment]
Rebased 👍

File: contrib/debian/dev-packages.lst (hunk 1 above)
Comment (adrianreber) [reply contrib/debian/dev-packages.lst:10]: I also think it should work if you remove python-ipaddr.

**Comment by adrianreber** [comment]
Looks good. CI is happy. Do you want to remove python-ipaddr (which indeed should not be needed with python 3)? From my point of view this can be merged with or without the removal of python-ipaddr.

**Comment by thaJeztah** [comment]
Let me try removing `python-ipaddr` from `dev-packages.lst`

**Review by rst0git — APPROVED** [review]

**Comment by thaJeztah** [comment]
removed; and CI still looks happy

**Review by adrianreber — APPROVED** [review]

**Pull request merged**